src/types/FilterMode.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterMode {
```

Comment:
The primary export of the file does not match the filename. Please rename the file to 'FilterMode.ts' (Current: 'file_name')